warehouse/bigquery/bigquery.go

```diff
@@ -591,8 +593,9 @@ func (bq *HandleT) LoadUserTables() (errorMap map[string]error) {
 			return
 		}
 	}
-
-	if !isDedupEnabled {
+	// set isDedupEnabled to true to enable dedup for all tables
```

Comment:
Can we also move this description where the configs are created as well? This will help people give more context to the descriptions of the config.

Reply:
Updated